README.md

```diff
@@ -83,13 +83,15 @@ import numpy as np
 from dhl_sdk import DataHowLabClient, APIKeyAuthentication
 
 # DHL_API_KEY env var is loaded from the .env file or added directly as an argument here 
-key = APIKeyAuthentication()
+key = APIKeyAuthentication("API_KEY")
 
 # This is an example. Change this line to your DataHowLab Instance
 your_url = "https://yourdomain.datahowlab.ch/"
-client = DataHowLabClient(auth_key=key, base_url=your_url)
+client = DataHowLabClient(auth_key=key, base_url=your_url, verify_ssl = True)
 ```
 
+*Note*: the parameter `verify_ssl` of `DataHowLabClient` controls whether the SSL certificates are verified (verification is enabled by default). Disabling SSL verification should only be done in controlled environments, such as on premise deployments with trusted servers.
```

Comment:
I would expand the documentation here to explain what the option does and how it is reflected on the underlying requests.

We could also provide some examples depending on the specific setup (e.g. SaaS, custom CA, self signed). In that case, it would probably require a dedicated section in the documentation.
I can help writing it, but let me know what's the best place where to put it, if directly in the README or somewhere else.